homeassistant/components/airgradient/entity.py

```diff
@@ -1,20 +1,31 @@
 """Base class for AirGradient entities."""
 
 from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from .const import DOMAIN
 from .coordinator import AirGradientCoordinator
 
 
-class AirGradientEntity(CoordinatorEntity[AirGradientCoordinator]):
+class AirGradientEntity(Entity):
     """Defines a base AirGradient entity."""
 
     _attr_has_entity_name = True
 
-    def __init__(self, coordinator: AirGradientCoordinator) -> None:
+    def __init__(self, serial_number: str) -> None:
         """Initialize airgradient entity."""
-        super().__init__(coordinator)
         self._attr_device_info = DeviceInfo(
-            identifiers={(DOMAIN, coordinator.serial_number)},
+            identifiers={(DOMAIN, serial_number)},
         )
+
+
+class AirGradientCoordinatedEntity(
```

Comment:
This change is not required as the update entity also uses a coordinator.
By reverting these changes the PR changes will be reduced

Reply:
But I don't want the update entity to write on every state update right?